public/app/features/alerting/unified/components/rule-viewer/RuleViewer.test.tsx

```diff
@@ -33,20 +40,58 @@ const ELEMENTS = {
       button: byRole('button', { name: /More/i }),
       actions: {
         silence: byRole('link', { name: /Silence/i }),
-        declareIncident: byRole('menuitem', { name: /Declare incident/i }),
         duplicate: byRole('menuitem', { name: /Duplicate/i }),
         copyLink: byRole('menuitem', { name: /Copy link/i }),
         export: byRole('menuitem', { name: /Export/i }),
         delete: byRole('menuitem', { name: /Delete/i }),
       },
+      pluginActions: {
+        sloDashboard: byRole('link', { name: /SLO dashboard/i }),
+        declareIncident: byRole('link', { name: /Declare incident/i }),
+        assertsWorkbench: byRole('link', { name: /Open workbench/i }),
```

Comment:
(Related: With https://github.com/grafana/grafana/pull/86886 we could force them to be menuitems, or it would depend on if the plugin was defining an onClick or a URL)

Reply:
> it would depend on if the plugin was defining an onClick or a URL

Sounds good to me